src/libraries/Microsoft.Extensions.Options/gen/Emitter.cs

```diff
@@ -484,32 +484,38 @@ public void EmitRangeAttribute(string modifier, string prefix, string className,
         public override string FormatErrorMessage(string name) =>
                 string.Format(global::System.Globalization.CultureInfo.CurrentCulture, GetValidationErrorMessage(), name, Minimum, Maximum);
         private bool NeedToConvertMinMax { get; }
-        private bool Initialized { get; set; }
+        private volatile bool _initialized;
         private const string c_minMaxError = "The minimum and maximum values must be set to valid values.";
```

Comment:
Nit: Could we make this follow our constant naming guidelines? i.e. MinMaxError rather than c_minMaxError

Reply:
Done.